src/components/PaintControls.vue

```diff
@@ -51,73 +51,128 @@
         </v-item>
       </v-item-group>
     </v-row>
-    <v-row no-gutters align="center">
-      <v-slider
-        v-if="mode === PaintMode.CirclePaint || mode === PaintMode.Erase"
-        :model-value="brushSize"
-        @update:model-value="setBrushSize"
-        density="compact"
-        hide-details
-        label="Size"
-        min="1"
-        max="50"
-      >
-        <template v-slot:append>
-          <v-text-field
-            :model-value="brushSize"
-            @input="setBrushSize"
-            variant="plain"
-            class="mt-n3 pt-0 pl-2"
-            style="width: 60px"
-            density="compact"
-            hide-details
-            type="number"
-            min="1"
-            max="50"
-          />
-        </template>
-      </v-slider>
-
-      <FillBetweenControls v-if="mode === PaintMode.FillBetween" />
-    </v-row>
+    <template v-if="mode === PaintMode.CirclePaint || mode === PaintMode.Erase">
+      <v-row no-gutters>Size (pixels)</v-row>
+      <v-row no-gutters align="center">
+        <v-slider
+          :model-value="brushSize"
+          @update:model-value="setBrushSize"
+          density="compact"
+          hide-details
+          min="1"
+          max="50"
+        >
+          <template v-slot:append>
+            <v-text-field
+              :model-value="brushSize"
+              @input="setBrushSize"
+              variant="underlined"
+              class="mt-n3 pt-0 pl-2 opacity-70"
+              style="width: 60px"
+              density="compact"
+              hide-details
+              type="number"
+              min="1"
+              max="50"
+            />
+          </template>
+        </v-slider>
+      </v-row>
+      <v-row no-gutters class="mb-1">Threshold </v-row>
+      <v-row v-if="currentImageStats" no-gutters align="center">
+        <v-range-slider
+          v-model="threshold"
+          :min="currentImageStats.scalarMin"
+          :max="currentImageStats.scalarMax"
+          :step="thresholdStepGranularity"
+        >
+          <template #prepend>
+            <v-text-field
```

Comment:
With the prostate, as I drag down the top handle, the text fields widths' jerks wider/narrower a tiny bit.  Happens when the appened text value drops a magnatude.